css/properties/display.json

```diff
@@ -871,6 +1071,279 @@
                 "deprecated": false
               }
             }
+          },
+          "xul_box_values": {
+            "__compat": {
+              "description": "<code>-moz-box</code> and <code>-moz-inline-box</code>",
+              "support": {
+                "chrome": {
+                  "version_added": false
+                },
+                "chrome_android": {
+                  "version_added": false
+                },
+                "edge": {
+                  "version_added": false
+                },
+                "edge_mobile": {
+                  "version_added": false
+                },
+                "firefox": {
+                  "version_added": true,
+                  "notes": "Scheduled for removal (See <a href='https://bugzil.la/879275>bug 879275)"
+                },
+                "firefox_android": {
+                  "version_added": true,
+                  "notes": "Scheduled for removal (See <a href='https://bugzil.la/879275>bug 879275)"
+                },
+                "ie": {
+                  "version_added": false
+                },
+                "opera": {
+                  "version_added": false
+                },
+                "opera_android": {
+                  "version_added": false
+                },
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": {
+                  "version_added": false
+                },
+                "samsunginternet_android": {
+                  "version_added": false
+                },
+                "webview_android": {
+                  "version_added": false
+                }
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": false,
+                "deprecated": true
+              }
+            }
+          },
+          "xul_grid_values": {
+            "__compat": {
+              "description": "<code>-moz-grid</code>, <code>-moz-inline-grid</code>, <code>-moz-grid-group</code> and <code>-moz-grid-line</code>",
+              "support": {
+                "chrome": {
+                  "version_added": false
+                },
+                "chrome_android": {
+                  "version_added": false
+                },
+                "edge": {
+                  "version_added": false
+                },
+                "edge_mobile": {
+                  "version_added": false
+                },
+                "firefox": {
+                  "version_added": true,
+                  "version_removed": "62",
+                  "notes": "See <a href='https://bugzil.la/1288572>bug 1288572"
```

Comment:
The bug is fixed. I don't think we need this note. (also, the markup is wrong/incomplete)